src/timepicker/timepicker.ts

```diff
@@ -63,17 +63,23 @@ const NGB_TIMEPICKER_VALUE_ACCESSOR = {
       <tr>
         <td>
           <input type="text" class="form-control" maxlength="2" size="2" 
-            [ngModel]="formatHour(model?.hour)" (ngModelChange)="updateHour($event)">
+            [ngModel]="formatHour(model?.hour)" (ngModelChange)="updateHour($event)"
+            (keyup.arrowUp)="changeHour(hourStep)" (keyup.arrowDown)="changeHour(-hourStep)"
+            (mouseup)="changeHour(hourStep)" (mousedown)="changeHour(-hourStep)">
```

Comment:
`mouseup` / `mousedown` are _not_ good events! We should be responding to mouse wheel event, not button clicks! See how it is done in angular-ui/bootstrap: https://github.com/angular-ui/bootstrap/blob/c61d16a9330c18105d074ffd23f14db06689809b/src/timepicker/timepicker.js#L207-L238

Applies to all the remaining fields (hours, minutes, seconds)